db/migrate/20180403144853_convert_vm_attrs_to_hash.rb

```diff
@@ -0,0 +1,57 @@
+class ConvertVmAttrsToHash < ActiveRecord::Migration[5.1]
+  def up
+    transform :to_h
+  end
+
+  def down
+    transform :to_params
+  end
+
+  def transform(transform_method)
+    say "Starting serialized attributes conversion, this can take long time based on data amount"
+    say "Converting Nics"
+    transform_batch_columns(Nic::Base, [:attrs, :compute_attributes], transform_method)
+    say "Converting Compute Attributes"
+    transform_batch_columns(ComputeAttribute, [:vm_attrs], transform_method)
+    say "Converting Compute Resources"
+    transform_batch_columns(ComputeResource, [:attrs], transform_method)
+    say "Converting Reports"
+    transform_batch_columns(Report, [:metrics], transform_method)
+    say "Converting Lookup Keys"
+    transform_batch_columns(LookupKey, [:default_value], transform_method)
+    say "Converting Lookup Values"
+    transform_batch_columns(LookupValue, [:value], transform_method)
+    say "All conversions finished"
+  end
+
+  def transform_batch_columns(base, serialized_columns, transform_method)
+    base.unscoped.find_each  do |object|
```

Comment:
Missing `.select(serialized_columns)`

Reply:
I can either use `find_each` or `select(serialized_columns)`, having both does not work.